### PR TITLE
feat: add PATCH and DELETE item membership mutations

### LINKED ISSUE #12: PATCH and DELETE item-membership

### PR DESCRIPTION
**Pull request by pyphilia**
close #12

### CODE REVIEW

File: src/mutations/index.ts [hunk 1]
```diff
@@ -2,13 +2,15 @@ import { QueryClient } from 'react-query';
 import itemMutations from './item';
 import memberMutations from './member';
 import tagsMutations from './itemTag';
+import itemMembershipsMutations from './membership';
```
Comment (juancarlosfarah) [inline src/mutations/index.ts:5]: Shouldn't this be `itemMembershipMutations`? (Singular `Membership`)

File: src/api/routes.ts [hunk 2]
```diff
@@ -2,6 +2,8 @@ import qs from 'qs';
 import { UUID } from '../types';
 
 export const ITEMS_ROUTE = 'items';
+export const ITEM_MEMBERSHIPS_ROUTE = 'item-memberships';
```
Comment (juancarlosfarah) [inline src/api/routes.ts:5]: Looks like you're mixing both `membership` and `memberships` for the same entity. We should probably make it consistent.

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Looks good! There's a minor thing about consistent naming, though. Please have a look.

File: src/api/routes.ts (hunk 2 above)
Comment (pyphilia) [reply src/api/routes.ts:5]: There might be some confusions, but usually it's `membership`, except for `GET` as it fetches multiple of them.

**Pull request merged**